app/controllers/api/v1/Web.java

```diff
@@ -1888,13 +1888,16 @@ public static JsonObject getExceptionDetailsJSON(String flowExecUrl) {
         long endTime = System.nanoTime();
         logger.info(" Processing of  "+ jobException.jobName +" took "+ (endTime - startTime) * 1.0 / (1000000000.0) + "s");
       }
+      logger.info(" Testing JSON Array "+jobsArray);
```

Comment:
Kindly log more intuitive comments.Also if possible use `debug` as this JSON can be large in many cases and occupy a lot of portion in the logs and act as an interruption between relevant logs.

Reply:
This should not been there , I have wrongly committed it . Removing it